### ISSUE TITLE
Build is broken?

### ISSUE THREAD

**Issue by laurentS** [hamaluik/timecop#35]
As of writing this issue, I have about 42 minutes of experience with flutter, apologies if I've missed something trivial.

I tried to setup the code to build locally, and couldn't get the app to build. I get a dependency resolution error:
```
$ flutter run
Because no versions of screenshots match >2.1.1 <3.0.0 and screenshots 2.1.1 depends on process ^3.0.9, screenshots ^2.1.1 requires process ^3.0.9.
And because every version of flutter_driver from sdk depends on process 4.0.0-nullsafety.2, screenshots ^2.1.1 is incompatible with flutter_driver from sdk.
So, because timecop depends on both flutter_driver any from sdk and screenshots ^2.1.1, version solving failed.
Running "flutter pub get" in timecop...
pub get failed (1; So, because timecop depends on both flutter_driver any from sdk and screenshots ^2.1.1, version solving failed.)
```
and I just installed flutter stable:
```
$ flutter --version
Flutter 1.22.4 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 1aafb3a8b9 (6 days ago) • 2020-11-13 09:59:28 -0800
Engine • revision 2c956a31c0
Tools • Dart 2.10.4
```

Based on a quick search, it looks like the dependencies and sdk versions are incompatible?

**Comment by hamaluik** [comment]
Should be fixed as of 091358078e6a9d29eac8cc53ac389040d8636a06.

Sorry about that!

**Comment by laurentS** [comment]
Cool, the build completes now, thanks for fixing it!
